### PR TITLE
Deploy scripts

### PR DESCRIPTION
**Pull request by nieznanysprawiciel**

### CODE REVIEW

File: build-image.sh [hunk 1]
```diff
@@ -5,12 +5,11 @@ mkdir -p repos/
...
-git clone git@github.com:golemfactory/gudot.git
-git clone git@github.com:golemfactory/key_cracker_cpp.git
-git clone git@github.com:golemfactory/key_cracker_demo.git
-git clone git@github.com:golemfactory/key_cracker_gen.git
-#git clone https://github.com/golemfactory/gwasm-runner.git
+./../prepare-git-repo.sh hello-gwasm-runner
+./../prepare-git-repo.sh gudot
+./../prepare-git-repo.sh key_cracker_cpp
+./../prepare-git-repo.sh key_cracker_demo
+./../prepare-git-repo.sh key_cracker_gen
```
Comment (tworec) [inline build-image.sh:12]: pls remove it. `key_cracker_gen` is on internal use only.

**Review by tworec — CHANGES_REQUESTED** [review]

**Comment by tworec** [comment]
@prekucki or @nieznanysprawiciel pls add here @mbenke PR
https://github.com/prekucki/gwasm-tutorial/pull/3

File: build-image.sh [hunk 2]
```diff
@@ -5,12 +5,11 @@ mkdir -p repos/
...
-git clone git@github.com:golemfactory/hello-gwasm-runner.git
-git clone git@github.com:golemfactory/gudot.git
-git clone git@github.com:golemfactory/key_cracker_cpp.git
-git clone git@github.com:golemfactory/key_cracker_demo.git
-git clone git@github.com:golemfactory/key_cracker_gen.git
-#git clone https://github.com/golemfactory/gwasm-runner.git
+./../prepare-git-repo.sh hello-gwasm-runner
+./../prepare-git-repo.sh gudot
+./../prepare-git-repo.sh key_cracker_cpp
+./../prepare-git-repo.sh key_cracker_demo
```
Comment (tworec) [inline build-image.sh:11]:
```suggestion
./../prepare-git-repo.sh key_cracker_rust
```
pls rename it to `_rust` as opossite to `_cpp`.
Whats more, there is contents of this repo duplicated within itself
```
root@d174bcaa9dda:~/key_cracker_cpp# ls -o
total 28
-rw-r--r-- 1 root   96 Oct  1 09:41 CMakeLists.txt
-rw-r--r-- 1 root   18 Oct  1 09:41 README.md
drwxr-xr-x 2 root 4096 Oct  1 10:39 bin
drwxr-xr-x 5 root 4096 Oct  1 10:39 build
drwxr-xr-x 4 root 4096 Oct  1 09:41 include
drwxr-xr-x 5 root 4096 Oct  1 09:52 key_cracker_cpp
drwxr-xr-x 2 root 4096 Oct  1 09:41 src
root@d174bcaa9dda:~/key_cracker_cpp# ls -o key_cracker_cpp/
total 16
-rw-r--r-- 1 root   96 Oct  1 09:52 CMakeLists.txt
-rw-r--r-- 1 root   18 Oct  1 09:52 README.md
drwxr-xr-x 4 root 4096 Oct  1 09:52 include
drwxr-xr-x 2 root 4096 Oct  1 09:52 src
```

And README.txt is empty....
Please at least copy wiki to README.txt
https://github.com/golemfactory/key_cracker_cpp/wiki

**Comment by tworec** [comment]
Also pls add welcome msg for the docker
https://docs.google.com/document/d/19ROy9z5dNWexGeyxErGwnaPkl575J07_6ZBM6WTzRGs/edit

http://ashgillman.github.io/2015/11/17/Docker-MOTD.html

```
Great that you’re here!

This is a toolset for the Golem workshop at deVcon, Osaka, Japan.
https://devcon.golem.network
Building on top of Golem with gWasm and Unlimited

You’ll find here:
- gcc
- emscripten sdk
- rust (with wasm32-unknown-emscripten target)
- python2, curl, vim and nano
- gwasm-runner (use -h to get help)

We had also prepared four examples for you:
  - hello world for gwasm runner (you’re now in its directory)
  - mandelbrot fractal example (~/mandelbrot) 
  - fully homomorphic encryption example (~/gudot)
  - easter egg: cracking challenge implemented in both Rust and C++ 

All of those can be compiled to WASM target and executed with gwasm-runner.
```

**Comment by nieznanysprawiciel** [comment]
All requested changes will be apllyed in this PR:
https://github.com/golemfactory/gwasm-tutorial/pull/4

**Comment by tworec** [comment]
> All requested changes will be apllyed in this PR:
> #4

so I'm closing this one

**Pull request closed without merging**